app/src/main/java/org/simple/clinic/enterotp/EnterOtpUpdate.kt

```diff
@@ -31,7 +32,7 @@ class EnterOtpUpdate(
   private fun effectsForStateChanged(model: EnterOtpModel, stateChanged: ProtectedState): Next<EnterOtpModel, EnterOtpEffect> {
     return when (stateChanged) {
       is Allowed -> next(model.setOtpEntryMode(stateChanged))
-      else -> // do something
+      is Blocked -> next(model.setOtpEntryMode(stateChanged))
```

Comment:
We can just extract out the state and just call `model.setOtpEntryMode` one time.